app/policies/application_policy.rb

```diff
@@ -70,13 +70,13 @@ def get_user_groups
     end
   end
 
-  def is_barito_superadmin?
+  def is_global_viewer?
     return false unless get_user_groups
-    get_user_groups.include?("barito-superadmin") 
+    get_user_groups.include?(Figaro.env.global_viewer_role)
   end
 
-  def is_global_viewer?
+  def barito_superadmin?
```

Comment:
Yes, I refactored the name. Reason is specified here: https://github.com/rubocop-hq/ruby-style-guide#bool-methods-prefix.

Reply:
okay, then why didn't we change is_global_viewer as well?

Reply:
I didn't change that because previously it's not part of team-based access. I change `barito_superadmin` one because it's involved on team-based access feature.

But if you want to, I can do this also.

Reply:
Anyway, `is_global_viewer` in `application_policy.rb` is not being used anywhere. 😅

Should I remove it?

Reply:
yep, please do

Reply:
IMO, you should change `is_global_viewer as well`, and also please do remove `is_global_viewer` in `application_policy` if it's not being used anywhere

cc: Mas @ekarisky

Reply:
I've removed it. Thanks!